authors/bruno-skvorc/polkadot/en_US/final.md

```diff
@@ -0,0 +1,186 @@
+
+Plans exist for adding more governance bodies, but for details on that I recommend reading the post linked above.
+
+It should be noted that making proposals for spending the Treasury's funds (see next section) requires a deposit of 5% of the target amount which is burned if the proposal does not pass, and that the Treasury must be spent entirely every period of 24 days, otherwise it has to burn some of its remaining funds. Both of these mechanics make for some deflationary pressure (see next section).
+
+## Transaction fees and treasuries
+
+In Ethereum, most of the [gas](https://bitfalls.com/2017/12/05/ethereum-gas-and-transaction-fees-explained/) spent on transactions is going to be [burned](https://eips.ethereum.org/EIPS/eip-1559). This creates deflationary pressure on the native currency, ether. The transactions being issued can contain a "tip" amount which acts as incentive for validators to prioritize certain transactions. This way, those using the network directly benefit all stakeholders as burned ether adds value to all ether.
+
+In Polkadot, transaction fees are split between the validators and the treasury, 20% - 80%. There is no burning. This is so that the protocol can throttle its inflation and deflation more effectively. As slashing is inherently an unpredictable event, burning the stake during mass-outage events would cause sudden deflation to no fault of the validators - it was an infrastructure or protocol level failure. In those cases, it's better to have the funds ready to pay for fixing things than to burn the funds and discourage network participation. The ratio of Treasury / Validator distribution can be changed through governance (see previous section).
+
+In Polkadot, blocks have reserved space for crucial transactions - like those of fishermen reporting misbehaviors. This allows fishermen to do their job even if the blocks are full.
+
+Like Ethereum, Polkadot also has tips on transactions for dealing with spikes in block space demand.
+
+## Consensus
+
+Ethereum implements finalization through a system called [Gasper](https://our.status.im/two-point-oh-justification-and-finalization/), a blend of LMD GHOST (Last Message Driven Greediest Heaviest Observed SubTree) and Casper FFG (friendly finality gadget). Finalization is the concept of being so certain a block is correct and cannot be undone that being wrong about this claim would result in the punishment of a large part of the network (a large number of validators). Polkadot uses GRANDPA (GHOST-based Recursive ANcestor Deriving Prefix Agreement) to do the same thing.
```

Comment:
> Finalization is the concept of being so certain a block is correct and cannot be undone that being wrong about this claim would result in the punishment of a large part of the network (a large number of validators)

I feel like this could be re-worded to be clearer.  The punishment is kind of a side effect, not part of the definition. Perhaps something like the definition in the Casper FFG paper, which should work for both Ethereum and Polkadot. https://arxiv.org/pdf/1710.09437v4.pdf